### PR TITLE
Give the docs package a little bit ❤️

### PR DESCRIPTION
**Pull request by paescuj**
A bit of linting & formatting, a bit of cleaning-up & updating...

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -1,41 +1,21 @@
...
-	"main": "dist/index.js",
-	"exports": {
-		".": "./dist/index.js",
-		"./package.json": "./package.json"
-	},
 	"scripts": {
 		"spellcheck": "spellchecker -p spell indefinite-article repeated-words syntax-mentions syntax-urls frontmatter",
-		"dev": "vitepress dev .",
-		"build": "vitepress build .",
-		"serve": "vitepress serve .",
```
Comment (paescuj) [inline docs/package.json:13]: Aligns with the (newer?) default of `vitepress`

File: docs/package.json [hunk 2]
```diff
@@ -1,41 +1,21 @@
...
-				"@types/react"
-			]
-		}
-	},
 	"dependencies": {
 		"node-fetch": "^3.3.1"
-	},
-	"packageManager": "pnpm@8.1.0",
-	"engines": {
-		"pnpm": ">=8.1.0"
```
Comment (paescuj) [inline docs/package.json:39]: Hopefully, `netlify` will use the `packageManager` field from the root.
I'd like to keep this in one place, if possible.
Of course we can revert that if we see that I won't work with `netlify`.

**Review by rijkvanzanten — APPROVED** [review]
Code changes LGTM

**Review by azrikahar — COMMENTED** [review]
Seems like VitePress `1.0.0-alpha.49` included https://github.com/vuejs/vitepress/pull/1989, so now our warning blocks would look like the following, with the gray background instead of colored background being the "bigger" difference:

|Light|Dark|
|---|---|
|||

vs our intended version (after #18117 gets released):

|Light|Dark|
|---|---|
|||

That said, we're still overriding the tip block with our primary color to look like the previous iteration:



Do we want to follow the new "colored border and text but gray background" by VitePress or override the gray background on our end?

**Comment by paescuj** [comment]
> Seems like VitePress `1.0.0-alpha.49` included [vuejs/vitepress#1989](https://github.com/vuejs/vitepress/pull/1989), so now our warning blocks would look like the following, with the gray background instead of colored background being the "bigger" difference:
> Light 	Dark
>  	
> 
> vs our intended version (after #18117 gets released):
> Light 	Dark
>  	
> 
> That said, we're still overriding the tip block with our primary color to look like the previous iteration:
> 
> 
> 
> Do we want to follow the new "colored border and text but gray background" by VitePress or override the gray background on our end?

Thanks @azrikahar!
Let's follow the design of VitePress 👍  Do we still need https://github.com/directus/directus/pull/18117 or should we revert that?

File: docs/.vitepress/components/ArticleFeedback.vue
Comment (licitdev) [inline docs/.vitepress/components/ArticleFeedback.vue:1]: Noticed some type warnings, the following change should resolve them.
However this change will update the payload values from `null` to `undefined` which shouldn't have any effect, unless the endpoint explicitly checks for `null`. 🤔 

```ts
const feedback: {
	id?: string;
	rating?: number;
	comments?: string;
} = reactive({});

...

@click="feedback.rating = undefined"
```

**Comment by azrikahar** [comment]
> Do we still need #18117 or should we revert that?

I believe we can keep it. Did a quick check and our change still makes the warning text contrast higher on light theme vs VitePress' default 👍 (and the background will continue to follow VitePress' new style)

File: docs/.vitepress/components/ArticleFeedback.vue [hunk 3]
```diff
@@ -89,9 +89,9 @@ async function handleSubmission() {
 				</div>
 				<div class="button-container">
 					<button
-						class="btn"
 						v-for="item in ratingOptions"
 						:key="item.value"
+						class="btn"
 						@click="
 							feedback.rating = item.value;
 							handleSubmission();
```
[the remark this answers is not in the document]
Comment (azrikahar) [reply docs/.vitepress/components/ArticleFeedback.vue]: resolved by https://github.com/directus/directus/pull/18140/commits/cbb8ffc286c4fb135daea3e0570cc754e9d04cbe

**Comment by paescuj** [comment]
> > Do we still need #18117 or should we revert that?
> 
> I believe we can keep it. Did a quick check and our change still makes the warning text contrast higher on light theme vs VitePress' default 👍 (and the background will continue to follow VitePress' new style)

Thanks! You're right, looks better with it 👍

File: docs/.vitepress/components/ArticleFeedback.vue
Comment (azrikahar) [reply docs/.vitepress/components/ArticleFeedback.vue:1]: Makes sense 👍 Updated in https://github.com/directus/directus/pull/18140/commits/a4702b3b2ae9d09070460fe8359727b154691f93.

**Review by azrikahar — APPROVED** [review]

**Pull request merged**